talk/owt/sdk/base/win/videorendererd3d11.h

```diff
@@ -91,6 +93,8 @@ class WebrtcVideoRendererD3D11Impl
   bool d3d11_raw_inited_ = false;
   // Remote view is using MPO swapchain.
   bool d3d11_mpo_inited_ = false;
+  // Support super resolution
+  bool allow_sr_ = false;

```

Comment:
The name seems to be confused, is SR allowed by user or driver? If it's allowed by user, it could be sr_enabled. If it's supported by driver, it could be sr_supported.

Reply:
done. updated to sr_enabled_.